### PR TITLE
Proofreading of the General section

### PR DESCRIPTION
**Pull request by jakejarvis**
I've started going through the docs to clean them up a bit and clarify some of the wording. Please let me know if I went too far or if any of my changes are incorrect, I'd be happy to revert them. If this looks good, I'll move on to the next section!

Thanks for a great product! ❤️

### CODE REVIEW

File: _docs/10_general/10_what-we-collect.md [hunk 1]
```diff
@@ -4,66 +4,67 @@ category: general
...
-> We **collect** and **store** device dimensions
+> We **do collect and store** device dimensions
 
-With the collection of dimensions of the browser window (`window.innerWidth`) we can show the most popular screen sizes. This is useful for you so you can make sure your website works great on all of those dimensions.
+Collecting the dimensions of a user's browser window (`window.innerWidth`) allows us to show you the most popular screen sizes. This is useful for making sure your website works great on all screens: phones, tablets, desktops, etc.
 
 ### Do Not Track
 
-> We skip when Do-Not-Track is enabled
+> We **do NOT collect or store** any data **if a visitor has Do Not Track** enabled
```
Comment (adriaandotcom) [inline _docs/10_general/10_what-we-collect.md]:
```suggestion
> By default we **do NOT collect or store** any data **if a visitor has Do Not Track** enabled
```

File: _docs/10_general/01_introduction.md [hunk 2]
```diff
@@ -4,10 +4,9 @@ category: general
 permalink: /
 ---
 
-In this documentation you will find most of our features explained and find instructions on how to enable certain features.
+Welcome to the Simple Analytics documentation! Here you will find detailed information on our features and how to customize them.
```
Comment (adriaandotcom) [inline _docs/10_general/01_introduction.md]: I like the [Stripe way](https://stripe.com/docs) of welcoming: "Whether you’re a startup or a global enterprise, learn how to integrate with Stripe to accept payments and manage your business online."

```suggestion
Whether you’re a startup or a global enterprise, learn how to integrate with Simple Analytics to move to privacy friendly analytics for your visitors without needed consent.
```

Something like this?

File: _docs/10_general/03_how-to-use-url-params.md [hunk 3]
```diff
@@ -30,18 +30,18 @@ https://example.com/?ref=android%3A%2F%2Fcom.example.app%2Fpath
...
 
-Referrer URLs contain the full URL that the browser gives us. It's something like `https://www.referrer.com/search?query=sensitive+information`. From this URL we store only `referrer.com` as the referrer and we keep `referrer.com/search` as the original referrer. We drop the query, protocol, and common subdomain.
+Referrer URLs contain the full URL that the browser gives us, such as `https://www.example.com/search?query=sensitive+information`. From this URL we store `referrer.example.com` as the referrer and keep `www.example.com/search` as the original URL. We drop the query (`?query=sensitive+information`), protocol (`https://`), and common subdomain (`www.`).
```
Comment (adriaandotcom) [inline _docs/10_general/03_how-to-use-url-params.md]: Fixed a few incorrect urls:

```suggestion
Referrer URLs contain the full URL that the browser gives us, such as `https://www.example.com/search?query=sensitive+information`. From this URL we store `example.com` as the referrer and keep `example.com/search` as the original URL. We drop the query (`?query=sensitive+information`), protocol (`https://`), and common subdomain (`www.`).
```

File: _docs/10_general/10_what-we-collect.md [hunk 4]
```diff
@@ -4,66 +4,67 @@ category: general
...
 ### Referrers
 
-> We **collect** and **partly store** referrer's
+> We **do collect** and **partially store** referrers
 
-Referrers answer the question _"Where did the visitor come from?"_. We have two ways of checking the source of a user that visited your website.
+Referrers answer the question _"Where did this visitor come from?"_. We have two ways of checking the source of a user visiting your website.
 
-First of all; With most requests, browsers send the URL of the previous website as a referrer. Second; website owners can add a URL param (`utm_source=...`) in the links they post elsewhere. The URL param can also be called `ref=...` or `source=...`. In case we get the referrer via the browser (with full URL) we store it the same as URLs (see above). You can find a list of the most popular referrers in your analytics dashboard.
+1. In most cases, browsers send the URL of the previous website as a referrer. We store the referrer the same as URLs (see above). You can find a list of the most popular referrers in your analytics dashboard.
```
Comment (adriaandotcom) [inline _docs/10_general/10_what-we-collect.md:57]: Way cleaner, nice!

**Review by adriaandotcom — CHANGES_REQUESTED** [review]
Thanks Jake! This is super kind of you. I went through the PR and added some comments. Love to have the extra eyes on our docs, thanks a lot. This is super cool.

File: _docs/10_general/03_how-to-use-url-params.md (hunk 3 above)
Comment (jakejarvis) [reply _docs/10_general/03_how-to-use-url-params.md]: whoops, good catch!

File: _docs/10_general/01_introduction.md (hunk 2 above)
Comment (jakejarvis) [reply _docs/10_general/01_introduction.md]: I like this too 👍

**Comment by jakejarvis** [comment]
It's the least I could do especially since I'm using the generous student pack plan right now!

File: _docs/10_general/01_introduction.md [hunk 5]
```diff
@@ -4,10 +4,9 @@ category: general
 permalink: /
 ---
 
-In this documentation you will find most of our features explained and find instructions on how to enable certain features.
+Whether you’re a startup or a global enterprise, learn how to integrate with Simple Analytics to move to privacy friendly analytics for your visitors without needed consent.
```
Comment (adriaandotcom) [inline _docs/10_general/01_introduction.md]: Should it be "needed consent" or "needing consent"?

Comment (adriaandotcom) [reply _docs/10_general/01_introduction.md]: Let me know what this should be @jakejarvis so I can merge your PR.

Comment (jakejarvis) [reply _docs/10_general/01_introduction.md]: Hmm, probably "needing consent" in that case but how does something like this sound split into two sentences?

"Whether you’re a startup or a global enterprise, learn how to integrate with Simple Analytics for privacy friendly analytics. No consent required from your visitors!"

Comment (adriaandotcom) [inline _docs/10_general/01_introduction.md]:
```suggestion
Whether you’re a startup or a global enterprise, learn how to integrate with Simple Analytics for privacy friendly analytics. No consent required from your visitors!
```

**Review by adriaandotcom — APPROVED** [review]
Perfect!

**Pull request merged**